app/src/styles/app.css

```diff
@@ -99,7 +99,7 @@ input.tm-field {
   padding: 7px;
   font-size: var(--sm);
   text-transform: uppercase;
-  top: 0;
+  top: 2px;
```

Comment:
And this change?

Reply:
the denom in TmField was not vertically centered